### DISCUSSION TITLE
question about the different color of blocks

### DISCUSSION THREAD

**Discussion by charlesselrahc421** [ablab/quast#165] [Q&A] [answered]
Dear QUAST team,

Thank you for your extraordinary works about this toolkit, which really helps me a lot with my current research field! 
But i still not so clear about the misassembled type with gray color blocks and red color blocks. I had tried to extract the relevant sequence of genome and reference genome from the result showed in the right panel, but i can't find the difference between the gray color blocks and red color blocks.
![截屏2021-04-20 下午2 48 53](https://user-images.githubusercontent.com/65232041/115350727-08612180-a1e8-11eb-8528-b029a315a8b9.png)

So, could you please explain me detailedly about the meaning of the gray color blocks and red color blocks?

Best wishes,

shaoyanhan

**Comment by alexeigurevich** [comment, marked as answer]
Hi Shaoyanhan,
Thanks for your feedback! Note that there is a color legend in the right bottom corner of the Icarus screen (right below the Contig info panel in your screenshot).
![image](https://user-images.githubusercontent.com/620893/115356595-772c8380-a1c4-11eb-912d-0624504fd7ad.png)

So, gray blocks are simply blocks of misassembled contigs with a type of misassembly which is currently unchecked (see "Show misassemblies" section with checkboxes in the top control panel). By default, local misassemblies are unchecked because they are usually not so critical compared to four other types of misassemblies. Feel free to check them and turn all gray blocks into red ones. This functionality is added to (1) simplify the search for specific misassembly events (e.g. you can uncheck all but one type of misassemblies and only the corresponding blocks will be red), (2) to highlight the difference between local and extensive (relocation, inversion, translocation) misassemblies for new users.

**Comment by charlesselrahc421** [comment]
Hi Alexeigurevich,
Thank you very much for your patience! 
It really useful, now i can figure out what is the "Unchecked" means, but i sincerely suggest that maybe you could add a description about the grey blocks especially the sentence you just noted "they are usually not so critical compared to four other types of misassemblies", which will really more clear as for a freshman like me.

Best wishes,
Shaoyanhan

**Comment by alexeigurevich** [comment]
Thanks for a nice suggestion! We will try to add in future releases. You may also always consult with our online [manual](http://quast.sourceforge.net/docs/manual.html) which has some explanation on the misassemblies and with the [QUAST paper](https://academic.oup.com/bioinformatics/article/29/8/1072/228832).
